packages/beacon-node/src/execution/engine/types.ts

```diff
@@ -268,6 +295,19 @@ export function parseExecutionPayload(
     (executionPayload as deneb.ExecutionPayload).excessBlobGas = quantityToBigint(excessBlobGas);
   }
 
+  if (ForkSeq[fork] >= ForkSeq.eip6110) {
+    const {depositReceipts} = data;
+    // Geth can also reply with null
+    if (depositReceipts == null) {
+      throw Error(
+        `depositReceipts missing for ${fork} >= eip6110 executionPayload number=${executionPayload.blockNumber} hash=${data.blockHash}`
+      );
+    }
+    (executionPayload as eip6110.ExecutionPayload).depositReceipts = depositReceipts.map((d) =>
+      deserializeDepositReceipts(d)
+    );
```

Comment:
Just do
```suggestion
    (executionPayload as eip6110.ExecutionPayload).depositReceipts = depositReceipts.map(deserializeDepositReceipts);
```
one less closure